pages/jobs/index.js

```diff
@@ -71,18 +72,19 @@ Join us!
 `;
   const jobPostUrl = encodeURIComponent(body);
   const hasPosts = Object.keys(posts).length;
+
+  const description = 'Browse through available job offers!';
+  const image = '/img/social/website-card.jpg'
```

Comment:
Instead of using the website-card as the image for this page, kindly add an image for respective pages here in the PR so that all the pages have proper images added in their `Head`. Make sure you add the images in the same folder with .webp extension.